### PR TITLE
Upgrade dtls v2

### PR DESCRIPTION
**Pull request by at-wat**
Follow dtls module structure update.

### Reference issue
Ref: https://github.com/pion/dtls/pull/185

### CODE REVIEW

File: certificate.go [hunk 1]
```diff
@@ -94,17 +94,21 @@ func (c Certificate) Expires() time.Time {
 // GetFingerprints returns the list of certificate fingerprints, one of which
 // is computed with the digest algorithm used in the certificate signature.
 func (c Certificate) GetFingerprints() ([]DTLSFingerprint, error) {
-	fingerprintAlgorithms := []dtls.HashAlgorithm{dtls.HashAlgorithmSHA256}
+	fingerprintAlgorithms := []string{"sha-256"}
```
Comment (Sean-Der) [inline certificate.go]: This not being an enum anymore is kind of unfortunate :/ I could see mistakes coming from this.

Comment (Sean-Der) [reply certificate.go]: But then exporting bloats the public API. I don't know which tradeoff is worse.

Comment (at-wat) [reply certificate.go]: We could add inverse func of https://github.com/pion/dtls/blob/master/pkg/crypto/fingerprint/hash.go. Having it would be good.

Comment (at-wat) [reply certificate.go]: https://github.com/pion/dtls/pull/186

Comment (Sean-Der) [reply certificate.go]: Change looks fantastic. I left my comments on other PR, but all approves from me :)

**Comment by Sean-Der** [comment]
@at-wat This is perfect! Thanks again for refactoring/cleaning stuff up. Not always the most fun work :(

**Pull request merged**